File: petstore-app/src/main/java/com/funtis/petstore/app/domain/Tag.java
package com.funtis.petstore.app.domain;

import javax.persistence.*;

/**
 * Created by Sławomir Kluz on 12/09/2017.
 */
@Entity
public class Tag {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(nullable = false)
    private String name;

    public Tag() {

    }

    public Tag(String name) {
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: petstore-mock/src/main/scala/utf/petstore/mock/model/Tag.scala
package utf.petstore.mock.model

/**
  * Created by Sławomir Kluz on 13/10/2017.
  */
case class Tag(id: Long, name: String)

object Tag {
  def someTags() = Seq(Tag(1, "red"), Tag(2, "lazy"), Tag(4, "sleepyhead"), Tag(3, "happy"))
}